altair/vegalite/v2/examples/interval_selection.py

```diff
@@ -0,0 +1,28 @@
+"""
+Interval Selection Example
+==========================
+
+This is an example of creating a stacked chart for which the domain of the
+top chart can be selected by interacting with the bottom chart.
+"""
+
+import altair as alt
+from vega_datasets import data
+sp500 = data.sp500()
+
+brush = alt.selection(type='interval', encodings=['x'])
+
+upper = alt.Chart().mark_area().encode(
+    x=alt.X('date', scale={'domain': brush.ref()}),
+    y='price'
+).properties(
+    width=600,
+    height=200
+)
+
+lower = upper.properties(
+    selection=brush,
```

Comment:
I wonder how would this `selection = brush` thing works when there are multiple selections on the same unit specs.

I wonder if there should be a `.add_selection(brush)` instead.

Reply:
If there are multiple selections, you can add them together, e.g. ``selection = brush + pan``.

Yes, an ``add_selection`` method is a good idea.